src/seedcase_sprout/core/path_local.py

```diff
@@ -0,0 +1,290 @@
+
+    Args:
+        resource_id: The ID of the resource.
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path to a specific resource in the data package.
+
+    Examples:
+        ```{python}
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package and resource structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            resource_path = Path(temp_path / "resources")
+            resource_path.mkdir()
+            sp.create_resource_structure(path=resource_path)
+
+            # Get the path to the resource
+            sp.path_resource(resource_id=1, path=temp_path)
+        ```
+    """
+    path = path_resources(path=path) / str(resource_id)
+    return check_is_resource_dir(path)
+
+
+def path_resource_data(resource_id: int, path: Path = Path.cwd()) -> Path:
+    """Gets the absolute path to a specific resource's data (i.e., Parquet) file.
+
+    Args:
+        resource_id: ID of the resource.
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path the specific resource's data file in a data package.
+
+    Examples:
+        ```{python}
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package and resource structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            # TODO: Update after writing data to resource
+            # resource_path = Path(temp_path / "resources")
+            # resource_path.mkdir()
+            # sp.create_resource_structure(path=resource_path)
+            # sp.write_resource_data_to_raw(
+            #   package_id=1,
+            #   resource_id=1,
+            #   data="path/to/data.csv")
+
+            # sp.write_resource_parquet(
+            #     raw_files=sp.path_resource_raw_files(resource_id=1, path=temp_path),
+            #     path=sp.path_resource_data(resource_id=1, path=temp_path))
+
+            # Get the path to the resource data
+            # sp.path_resource_data(resource_id=1, path=temp_path)
+        ```
+    """
+    path = path_resource(resource_id, path=path) / "data.parquet"
+    return check_is_file(path)
+
+
+def path_resource_raw(resource_id: int, path: Path = Path.cwd()) -> Path:
```

Comment:
Following the discussion in #1078 that lead to this issue #1127 this should probably also be renamed:
```suggestion
def path_resource_batch(resource_id: int, path: Path = Path.cwd()) -> Path:
```